src/components/dashboard/DepositTokens/DepositTokensForm.scss

```diff
@@ -29,6 +29,9 @@
     display: flex;
     flex-direction: column;
   }
+  > span:first-child {
+    margin-left: 0;
+  }
   .action {
```

Comment:
decouple `.actions` and `.action`.
move `margin-left: 0` for `:first-child` from `includes` to upper level of `.action`